doc/source/management/backup_locks.rst

```diff
@@ -9,7 +9,7 @@
 ``LOCK TABLES FOR BACKUP``
 ---------------------------
 
-``LOCK TABLES FOR BACKUP`` uses a new MDL lock type to block updates to non-transactional tables and DDL statements for all tables. More specifically, if there's an active ``LOCK TABLES FOR BACKUP`` lock, all DDL statements and all updates to MyISAM, CSV, MEMORY and ARCHIVE tables will be blocked in the ``Waiting for backup lock`` status as visible in ``PERFORMANCE_SCHEMA`` or ``PROCESSLIST``. ``SELECT`` queries for all tables and ``INSERT/REPLACE/UPDATE/DELETE`` against |InnoDB|, Blackhole and Federated tables are not affected by ``LOCK TABLES FOR BACKUP``. Blackhole tables obviously have no relevance for backups, and Federated tables are ignored by both logical and physical backup tools.
+``LOCK TABLES FOR BACKUP`` uses a new MDL lock type to block updates to non-transactional tables and DDL statements for all tables. More specifically, if there's an active ``LOCK TABLES FOR BACKUP`` lock, all DDL statements and all updates to MyISAM, CSV, MEMORY and ARCHIVE tables will be blocked in the ``Waiting for backup lock`` status as visible in ``PERFORMANCE_SCHEMA`` or ``PROCESSLIST``. ``SELECT`` queries for all tables and ``INSERT/REPLACE/UPDATE/DELETE`` against |InnoDB|, |MyRocks|, |TokuDB|, Blackhole and Federated tables are not affected by ``LOCK TABLES FOR BACKUP``. Blackhole tables obviously have no relevance for backups, and Federated tables are ignored by both logical and physical backup tools. Only ``SELECT ...`` statements are allowed on |MyRocks| and |TokuDB| tables. Updating or inserting data as well as  DDLs are blocked.
```

Comment:
In this part:
"``SELECT`` queries for all tables and ``INSERT/REPLACE/UPDATE/DELETE`` against |InnoDB|, |MyRocks|, |TokuDB|, Blackhole and Federated tables are not affected by ``LOCK TABLES FOR BACKUP``."
I don't think MyRocks and TokuDB should be here and also this seems conflicting with the last 2 sentences in this paragraph.
I have made a comment in the Jira ticket with examples.
But I'll let George be the judge on this one.

Reply:
Agreed, these are conflicting statements:

> if there's an active LOCK TABLES FOR BACKUP ...
> INSERT/REPLACE/UPDATE/DELETE against ... MyRocks, TokuDB ... are not affected by 
> Only SELECT ... statements are allowed on MyRocks and TokuDB

I believe this should all be worded as:

> ``LOCK TABLES FOR BACKUP`` uses a new MDL lock type to block updates to non-transactional tables and DDL statements for all tables. More specifically, if there's an active LOCK TABLES FOR BACKUP lock, all DDL statements and all updates to MyISAM, CSV, MEMORY, ARCHIVE, |TokuDB|, and |MyRocks| tables will be blocked in the ``Waiting for backup lock`` status as visible in ``PERFORMANCE_SCHEMA`` or ``PROCESSLIST``. ``SELECT`` queries for all tables and ``INSERT/REPLACE/UPDATE/DELETE`` against |InnoDB|, Blackhole and Federated tables are not affected by ``LOCK TABLES FOR BACKUP``. Blackhole tables obviously have no relevance for backups, and Federated tables are ignored by both logical and physical backup tools.

Reply:
Ready for review. The new version hopes to be delivering the same information claiming to have improved the readability :)

Reply:
I think this clarifies things and is more accurate, @tplavcic what say you?